code/DDSCodeTester.cpp

```diff
@@ -647,6 +647,39 @@ void dds_domain_examples()
                 "HISTORY_LATENCY_TOPIC;ACKNACK_COUNT_TOPIC;DISCOVERY_TOPIC;PHYSICAL_DATA_TOPIC");
         //!--
     }
+    {
+        // FASTDDS_MONITOR_SERVICE_PROPERTY
+        DomainParticipantQos pqos;
+
+        // Activate Fast DDS Monitor Service through properties
+        pqos.properties().properties().emplace_back("fastdds.enable_monitor_service",
+                "true");
+
```

Comment:
Include the other way to enable monitor service through the statistics

```suggestion
        // Activate Fast DDS Monitor Service through properties
        pqos.properties().properties().emplace_back("fastdds.enable_monitor_service",
                "true");
                
        // Activate Fast DDS Monitor Service through statistics properties (other way)
        pqos.properties().properties().emplace_back("fastdds.statistics",
                "MONITOR_SERVICE_TOPIC");

```